client/components/term-picker-option/style.scss

```diff
@@ -0,0 +1,77 @@
+.term-picker-option {
+  $padding-size: 18px;
+
+  display: flex;
+  width: 100%;
+  cursor: pointer;
+
+  padding: $padding-size;
+  border-radius: 5px;
+
+  background: $white;
+
+  .term-picker-option__radio-wrapper {
+    width: 21px;
+    margin-right: 12px;
+  }
+
+  .term-picker-option__radio {
+    width: 21px;
+    height: 21px;
+
+    &:checked:before {
+      width: 13px;
+      height: 13px;
+    }
+  }
+
+  .term-picker-option__header,
+  .term-picker-option__description {
+    width: 100%;
+  }
+
+  .term-picker-option__term,
+  .term-picker-option__price,
+  .term-picker-option__discount,
+  .term-picker-option__side-note,
+  .term-picker-option__description {
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  .term-picker-option__header {
+    line-height: 24px;
+  }
+
+  .term-picker-option__term {
+    display: inline-block;
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .term-picker-option__discount {
+    margin-left: 10px;
+  }
+
+  .term-picker-option__description {
+    font-size: 16px;
+  }
+
+  .term-picker-option__price {
+    color: $gray-dark;
+  }
+
+  .term-picker-option__side-note {
+    color: $gray;
+    margin-left: 5px;
+  }
+
+  &:not( .term-picker-option--active ) {
+    border: 1px solid $gray;
```

Comment:
This gray border looks a lot darker than the mockup. I'd suggest looking at the `Card` component or other similar components and grabbing the same border color.